src/core/services/layout/layout.scss

```diff
@@ -112,8 +112,8 @@
     display: -ms-flexbox;
     display: flex;
   }
-  .layout#{$name}-column {  flex-direction: column;  }
-  .layout#{$name}-row    {  flex-direction: row;     }
+  .layout#{$name}-column {  flex-direction: column; flex-shrink: 0;  }
```

Comment:
does not appear to change anything. But I also can not validate its value. I will redact this for now.